tests/dhcp_relay/test_dhcp_relay.py

```diff
@@ -96,8 +96,21 @@ def dut_dhcp_relay_data(duthost, ptfhost, testbed):
 
     return dhcp_relay_data_list
 
+@pytest.fixture(scope="module")
+def validate_dut_routes_exist(duthost, dut_dhcp_relay_data):
+    """Fixture to valid a route to each DHCP server exist
+    """
+    dhcp_servers = set()
+    for dhcp_relay in dut_dhcp_relay_data:
+        dhcp_servers |= set(dhcp_relay['downlink_vlan_iface']['dhcp_server_addrs'])
+
+    for dhcp_server in dhcp_servers:
+        result = duthost.shell("ip route get {0}".format(dhcp_server))
```

Comment:
can you reuse this function? duthost.get_ip_route_info(ipaddress.ip_address(u"0.0.0.0"))

example here.

https://github.com/Azure/sonic-mgmt/blob/master/tests/route/test_default_route.py#L31

Reply:
Done